### PR TITLE
ModelService

### PR DESCRIPTION
**Pull request by murdocb**
As i mentioned in the issue. Here is the pull request for a Service based on a ModelForm for easy validation with a model.

### CODE REVIEW

File: service_objects/models.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
```
Comment (c17r) [inline service_objects/models.py]: This shouldn't be here.  It's a test model but you've stuck it inside service_objects/ which is what gets packaged and distributed.  If you need a model for your testing, you should create it in tests/test_services.py

File: service_objects/services.py [hunk 2]
```diff
@@ -19,3 +19,20 @@ def service_clean(self):
 
     def process(self):
         raise NotImplementedError()
+
+
+class ModelService(forms.ModelForm):
```
Comment (c17r) [inline service_objects/services.py]: Basing ModelService off ModelForm comes with some baggage of ModelForms functionality tied tightly to it controlling a model's create/update/delete cycle.  Plus, this implementation duplicates everything between Service and ModelService.

A better approach would be:

```python
from django.forms import models

class ModelService(Service, metaclass=models.ModelFormMetaclass):
	pass

```
`ModelFormMetaclass` metaclass is what gives the functionality of looking at a model and auto-magically creating fields from it.  That's really the only functionality you need for ModelService.  And this way, changes to Service is reflected in ModelService without duplication.

File: compilemessages.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import os
```
Comment (c17r) [inline compilemessages.py]: The language changes should be split out into a separate PR since they have no bearing on ModelService.

**Comment by c17r** [comment]
Commit 5afe0c9c implements ModelService in the preferred style.  The 3 commits involving language changes have been cherry-picked in so credit is given where credit is due, thanks for setting that up!

Closing the PR.

**Comment by murdocb** [comment]
@c17r Thanks! I've been very busy this past week. Returning again now.
